samples/HttpRequestFeaturesv2/ClientApp/components/home-page/home-page.html

```diff
@@ -0,0 +1,48 @@
+<p> </p>
+<h2>Loading this app:</h2>
+<ul>
+    <li>Displays user agent information extracted from the http request.</li>
+    <li>Creates a personalizer rank request with that information, as well as randomized user taste info.</li>
+    <li>Creates a reward request with a reward value of 0.</li>
+</ul>
+<h2>Interacting with this page:</h2>
+<ul>
+    <li>To send the rank request, press the "Send request" button above the rank request body.</li>
+    <li>Once you do so, the rank response will be displayed below the request.</li>
+    <li>After sending a rank, you can send a corresponding reward. Use the "Send Request" button above the reward request body.</li>
+    <li>You can generate a new request with different taste info at any time by pressing the "Generate request" button next to Rank Request. 
+    This will reset the rank and reward responses to be empty.</li>
+</ul>
```

Comment:
This page is has quite a few changes. Please have it reviewed by someone on the PM team before submitting.

Reply:
I've pinged Ed for his review

Reply:
@limotley  Please Ping Amit if Ed is unavailable.

Reply:
I've pinged both, hopefully one should review